### ISSUE TITLE
Have a horizontal scrollbar at the top of the calendar in timeline view

### ISSUE THREAD

**Issue by dstromberg2** [fullcalendar/fullcalendar#4840]
Reduced test case:
https://codepen.io/anon/pen/qwBKzb

Reproduction steps:
Hover mouse in the events grid body, and use any scroll wheel action (or two finger swipe on an apple device within the body). Then click on the top scrollbar and try to drag either direction.

Result:
The headers will scroll, but the event grid body will not scroll in sync with it.

Investigation:
In ScrollJoiner.ts, for any regular scroll action, the current scrolling element is bound as the 'masterScroller' to broadcast its scroll position changes. On scrollEnd event, the masterScroller is unassigned.
However, the 'mousewheel' and related events bind the current scrolling element as the 'masterScroller', but have no action to release the assignment of masterScroller. Therefore, once a mousewheel event is registered, no other element is able to act as master.

**Comment by Goddard** [comment]
Yes same issue here.  I tried the stackoverflow suggestion, but this just inverted all the text in the div as well.  Perhaps I didn't implement it properly?
